File: Angular/app/Commandes/Models/CommandeUpdateParameters.ts
export class CommandeUpdateParameters {


    constructor() {

        this.Id = null;

        this.StatutId = null;
        this.StatutLibelle = null;

        this.ReferenceTransaction = null;
        this.ReferenceExterne = null;

    }


    public Id: number | null;

    public StatutId: number | null;
    public StatutLibelle: string | null;

    public ReferenceTransaction: string | null;
    public ReferenceExterne: string | null;

    public EleveId: number | null;

}

File: WS/Models/IN/CommandeUpdateParameters.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WS.Models.IN
{

    public class CommandeUpdateParameters
    {

        public Int32? Id { set; get; }

        public String ReferenceTransaction { set; get; }

        public String ReferenceExterne { set; get; }

        public Int32? StatutId { set; get; }

        public String StatutLibelle { set; get; }

        public Int32? EleveId { set; get; }

        public String TrackingNumber { set; get; }

    }

}